server/src/main/scala/com/cloudera/livy/server/ApiVersioningSupport.scala

```diff
@@ -0,0 +1,98 @@
+package com.cloudera.livy.server
+
+import javax.servlet.http.HttpServletRequest
+
+import org.scalatra.{NotAcceptable, ScalatraBase}
+
+/**
+  * Livy's servlets can mix-in this trait to get API version support.
```

Comment:
As long as the comment is in `/** */` Javadoc will parse it just fine. Whitespaces don't matter to the parser.
Unfortunately Intellij doesn't offer an option to switch. It's following the recommended style of Scaladoc. Which is compatible with Javadoc.